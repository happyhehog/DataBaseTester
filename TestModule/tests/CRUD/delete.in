[@Clear]
[@PrintLevel] MAIN

@@ ==============
@@ Testing DELETE
@@ ==============

[@NoOutput]
CREATE TABLE t(id INT NOT NULL PRIMARY KEY, height FLOAT, nickname CHAR(20), num INT);

[#Repeat] 20
INSERT INTO t VALUES ($i, $i, $i, $i);

DELETE FROM t WHERE id = 1;
DELETE FROM t WHERE id >= 18 and id <= 18;
DELETE FROM t WHERE height = 15 and id = 15;

[@NoOutput]

SELECT * from t;

[@NoOutput]
DROP TABLE t;
[@NoOutput]