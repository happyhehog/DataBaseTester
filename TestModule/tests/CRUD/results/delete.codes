{"status":"OK","command":"SELECT * from t;","answer":"id|height   |nickname|num|\n2 |2.000000 |'2'     |2  |\n3 |3.000000 |'3'     |3  |\n4 |4.000000 |'4'     |4  |\n5 |5.000000 |'5'     |5  |\n6 |6.000000 |'6'     |6  |\n7 |7.000000 |'7'     |7  |\n8 |8.000000 |'8'     |8  |\n9 |9.000000 |'9'     |9  |\n10|10.000000|'10'    |10 |\n11|11.000000|'11'    |11 |\n12|12.000000|'12'    |12 |\n13|13.000000|'13'    |13 |\n14|14.000000|'14'    |14 |\n16|16.000000|'16'    |16 |\n17|17.000000|'17'    |17 |\n19|19.000000|'19'    |19 |\n20|20.000000|'20'    |20 |"}
