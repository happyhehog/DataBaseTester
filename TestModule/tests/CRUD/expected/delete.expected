id|height   |nickname|num|
2 |2.000000 |'2'     |2  |
3 |3.000000 |'3'     |3  |
4 |4.000000 |'4'     |4  |
5 |5.000000 |'5'     |5  |
6 |6.000000 |'6'     |6  |
7 |7.000000 |'7'     |7  |
8 |8.000000 |'8'     |8  |
9 |9.000000 |'9'     |9  |
10|10.000000|'10'    |10 |
11|11.000000|'11'    |11 |
12|12.000000|'12'    |12 |
13|13.000000|'13'    |13 |
14|14.000000|'14'    |14 |
16|16.000000|'16'    |16 |
17|17.000000|'17'    |17 |
19|19.000000|'19'    |19 |
20|20.000000|'20'    |20 |